metron-deployment/packaging/ambari/metron-mpack/pom.xml

```diff
@@ -60,6 +64,19 @@
                             <failIfNoMatch>false</failIfNoMatch>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>metron-short-version</id>
+                        <goals>
+                            <goal>regex-property</goal>
+                        </goals>
+                        <configuration>
+                            <name>metron.short.version</name>
+                            <value>${project.parent.version}</value>
+                            <regex>\.</regex>
```

Comment:
Hm, we haven't done a SNAPSHOT release version before. Agreed we need to handle hyphens, for instance. Any idea how Ambari handles it in the service advisor class name? Can we have "SNAPSHOT" in there?